examples/jf-admin-app/linux/JFAManager.cpp

```diff
@@ -84,6 +98,46 @@ void JFAManager::HandleCommissioningCompleteEvent()
     }
 }
 
+bool JFAManager::IsDeviceJFAdmin()
+{
+    if (jfFabricIndex == kUndefinedFabricId)
+    {
+        return false;
+    }
+
+    CATValues cats;
+
+    if (mServer->GetFabricTable().FetchCATs(jfFabricIndex, cats) == CHIP_NO_ERROR)
```

Comment:
should this first check if this is the JFAnchor()?

does the anchor also need the JF Admin tag? Seems like it would be overkill to require both on the same NOC.